CBLClient/Apps/CBLTestServer-iOS/CBLTestServer-iOS/Server/PeerToPeerRequestHandler.swift

```diff
@@ -178,3 +205,157 @@ extension PeerToPeerRequestHandler: MessageEndpointDelegate {
     }
 }
 
+private class LocalWinCustomConflictResolver: ConflictResolverProtocol {
+    func resolve(conflict: Conflict) -> Document? {
+        let localDoc = conflict.localDocument!
+        let remoteDoc = conflict.remoteDocument!
+        let docId = conflict.documentID
```

Comment:
need to make sure either the local doc or the remote doc is not null, if any of the doc happens to be a null object, then this is not appropriate scenario, it may also cause crash when getting id out of it.

Reply:
same as above

Reply:
Added null check